docs/current/cookbook.md

```diff
@@ -453,6 +453,31 @@ The following code listing defines various environment variables for build purpo
 
 [Learn more](./guides/648384-multi-builds.md)
 
+### Reuse container images across pipelines
+
+The following code listing builds a container image in one pipeline and reuses it in another. This is useful if, for example, you need to create a self-contained base image in one pipeline and then use it for other operations (test, build, deploy...) in other pipelines.
```

Comment:
From this explanation alone it's not necessary to go through `id()`, just pass around the object instance. For the use case in the examples it would have to be something from two different runs or dagger sessions.

Reply:
The recipe is based on the original Discord thread by @marcosnils https://discord.com/channels/707636530424053791/708371226174685314/1155137294538375198. That listing doesn't involve separate Dagger sessions, though. 

Just to check that I understood correctly, you mean that the example shown in the Discord thread can also be achieved without id(), by passing the container object?

Reply:
Ah, that's something else. My confusion was due to the general term "pipeline" vs the very specific `Container.pipeline` field. It's a workaround though, because when you use `Pipeline()` it creates a sub-group, and this is a way to not do that. That's the reason for having https://github.com/dagger/dagger/issues/5303.

Can you make this use case clearer?